Header header

uint32 image_seq_
time image_stamp_

uint32 most_active_cell_

uint32[] cell_id_
float64[] cell_rate_
